dotcom-rendering/scripts/webpack/webpack.config.browser.js

```diff
@@ -89,44 +90,74 @@ const getLoaders = (bundle) => {
 					},
 				},
 			];
+		case 'apps':
```

Comment:
Is the term `browser` for this file getting slightly overloaded and congested if it also includes apps.

This doesn't have to be now but could the 'apps' flow be separated into its own `webpack.config.apps.js` ?

Reply:
Hmmm, I'll take a think about this, though I do have a PR open to rename browser to client ... https://github.com/guardian/dotcom-rendering/pull/7125